### PR TITLE
[TACHYON-1446] Make a client Utils class

### PR DESCRIPTION
**Pull request by calvinjia**
https://tachyon.atlassian.net/browse/TACHYON-1446

### CODE REVIEW

**Comment by yupeng9** [comment]
LGTM

File: clients/unshaded/src/main/java/tachyon/client/ClientContext.java [hunk 1]
```diff
@@ -111,14 +111,6 @@ public static synchronized InetSocketAddress getMasterAddress() {
     return sMasterAddress;
   }
 
-  /**
-   * @return a random non-negative long
-   */
-  public static synchronized long getRandomNonNegativeLong() {
-    checkContextInitialized();
-    return Math.abs(sRandom.nextLong());
```
Comment (gpang) [inline clients/unshaded/src/main/java/tachyon/client/ClientContext.java]: is the sRandom variable still used elsewhere in this class?

File: clients/unshaded/src/main/java/tachyon/client/Utils.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import java.util.Random;
+
+/**
+ * Utilities class for Tachyon Client. All methods and variables are static. This class is thread
+ * safe.
+ */
+public final class Utils {
+  private static Random sRandom = new Random();
+
+  public static synchronized long getRandomNonNegativeLong() {
```
Comment (gpang) [inline clients/unshaded/src/main/java/tachyon/client/Utils.java]: add javadoc

**Comment by gpang** [comment]
Just some minor comments. Thanks!

File: clients/unshaded/src/main/java/tachyon/client/ClientContext.java (hunk 1 above)
Comment (calvinjia) [inline clients/unshaded/src/main/java/tachyon/client/ClientContext.java]: Good catch, thanks

File: clients/unshaded/src/main/java/tachyon/client/Utils.java (hunk 2 above)
Comment (calvinjia) [inline clients/unshaded/src/main/java/tachyon/client/Utils.java]: Updated, thanks.

**Comment by calvinjia** [comment]
@gpang I've updated the PR, could you take a look?

**Comment by gpang** [comment]
LGTM

**Pull request merged**